.goreleaser.yml

```diff
@@ -29,14 +29,14 @@ dockers:
       - "trussworks/ecs-service-logs:{{ .Tag }}"
     skip_push: true
 archives:
-- replacements:
-    darwin: Darwin
-    linux: Linux
-    windows: Windows
-    386: i386
-    amd64: x86_64
+  -
+    replacements:
+      darwin: Darwin
+      linux: Linux
+      amd64: x86_64
 checksum:
   name_template: 'checksums.txt'
+  algorithm: sha256
```

Comment:
Is this for upstream dependency mismatches? Does dependabot not do this dirty work for us?

Reply:
It does do this but I think its important to call this out explicitly instead of hoping folks implicitly expect it or having it change underneath us in the event goreleaser does some update to their defaults.

Reply:
it's really more of a best practice when releasing software on the internet. It's good to create the checksums so that folks that download things can double check the file they downloaded matches what we say it should be.

sha256 is dfinitely better than md5 sums.... which was the old standard. that's all it really is.... goreleaser will publish. the sha256 sum.

dependabot doesn't really have anything to do with this config.